types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
+     */
+    name: string 
+    /**
+     * camillebeaumont name and description fix,channel description
+     */
+    description: string
+    /**
+     * channel importance level
+     */
+    importance: integer
+    /**
```

Comment:
We're missing the `sound` attribute on the `Channel` object interface. We might as well use the opportunity to make the whole interface documented.
```suggestion
    /**
     * The name of the sound file to be played upon receipt of the notification in this channel. Cannot be changed after channel is created.
     */
    sound: string
    /**
```